### PR TITLE
console: update to tectonic-console-builder:v20

### PR DESCRIPTION
**Pull request by dtaylor113**

### CODE REVIEW

File: projects/origin-stable/release.yaml [hunk 1]
```diff
@@ -65,8 +65,8 @@ spec:
     from: {"kind":"DockerImage","name":"docker.io/openshift/origin-release:nodejs-8-browser-tests"}
     referencePolicy: {"type": "Local"}
     importPolicy: {"scheduled": true}
-  - name: tectonic-console-builder-v19
-    from: {"kind":"DockerImage","name":"quay.io/coreos/tectonic-console-builder:v19"}
```
Comment (spadgett) [inline projects/origin-stable/release.yaml:69]: Don't remove v19, that will break release-4.4 PRs

Comment (dtaylor113) [reply projects/origin-stable/release.yaml:69]: Ok, do I add a new one for v20 then?

Comment (spadgett) [reply projects/origin-stable/release.yaml:69]: > Ok, do I add a new one for v20 then?

Yes, v19 will be used for 4.4, and v20 will be used for 4.5+. So we need both.

Comment (dtaylor113) [reply projects/origin-stable/release.yaml]: ok, fixed, ty

**Comment by spadgett** [comment]
@openshift/openshift-team-developer-productivity-test-platform for approval

**Comment by petr-muller** [comment]
/approve
/hold

Holding if you want to wait for rehearsals, unhold at will

**Comment by spadgett** [comment]
Thanks @petr-muller 

None of the test failures I see seem to be caused by the PR, and the previous runs overall look good. Removing the hold.

/hold cancel

**Pull request merged**